app/questionnaire/placeholder_parser.py

```diff
@@ -142,19 +147,24 @@ def _parse_transforms(
 
         for transform in transform_list:
             transform_args: MutableMapping = {}
+            value_source_resolver = self._get_value_source_resolver_for_transform(
```

Comment:
What is the purpose of TRANSFORMS_REQUIRING_ROUTING_PATH? We only want to initialise new VSR when certain transform is being processed?

Reply:
Removed, was meant to be removed before just forgot . Should be evaluated at questionnaire schema 👍